web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/TokenV2Controller.java

```diff
@@ -0,0 +1,90 @@
+package it.pagopa.selfcare.dashboard.web.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import it.pagopa.selfcare.commons.base.logging.LogUtils;
+import it.pagopa.selfcare.commons.base.security.SelfCareUser;
+import it.pagopa.selfcare.commons.web.security.JwtAuthenticationToken;
+import it.pagopa.selfcare.dashboard.web.model.ExchangedToken;
+import it.pagopa.selfcare.dashboard.web.model.IdentityTokenResource;
+import it.pagopa.selfcare.dashboard.web.security.ExchangeTokenService;
+import it.pagopa.selfcare.dashboard.web.security.ExchangeTokenServiceV2;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/v2/token", produces = MediaType.APPLICATION_JSON_VALUE)
+@Api(tags = "token")
+public class TokenV2Controller {
+
+    private final ExchangeTokenServiceV2 exchangeTokenService;
+
+
+    @Autowired
+    public TokenV2Controller(ExchangeTokenServiceV2 exchangeTokenServiceV2) {
+        this.exchangeTokenService = exchangeTokenServiceV2;
+    }
+
+
+    @GetMapping(value = "exchange")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.dashboard.token.api.exchange}")
```

Comment:
As mentioned in previous pull requests, we should add a specific operation identifier for the endpoints